README.md

```diff
@@ -1,3 +1,20 @@
 ## Welcome to the .NET SDK Workload Versions repo
 
-This repository contains the version information for .NET SDK Workloads.
\ No newline at end of file
+This repository contains the version information for .NET SDK Workloads.
+
+### Pre-requisites for local VS insertion build
+
+1. Install the latest [Visual Studio](https://visualstudio.microsoft.com/downloads/) with the .NET Desktop workload
+  - Make sure to restart your PC after the installation is complete.
+2. [Install Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli-windows#install-or-update)
+3. Run `az login` to authenticate
+  - When it asks for a subscription to select, just press Enter. The default subscription selection does not affect DARC.
+4. [Install DARC](https://github.com/dotnet/arcade/blob/main/Documentation/Darc.md#setting-up-your-darc-client)
+5. [Add GitHub auth for DARC](https://github.com/dotnet/arcade/blob/main/Documentation/Darc.md#step-3-set-additional-pats-for-azure-devops-and-github-operations)
```

Comment:
Do you need al of the darc auth details here or just link to arcade (or the details are included when you run darc authenticate)?

Reply:
I think the 3 bullet points below this are to convey the exact actions you need to take. Skimming through the generic docs can take time to grok. These steps are almost exactly what I did, and for people who never used DARC would probably want something a bit more straightforward to simply get set up.